src/styles/card.scss

```diff
@@ -0,0 +1,155 @@
+.card {
+  box-sizing: border-box;
+  width: $cardWidth;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    padding: 32px 19px 36px;
+  }
+
+  &__photo {
+    display: inline-block;
+    width: 100%;
+    height: 134px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: nowrap;
+    padding: 0 16px 24px;
+    width: 100%;
+    box-sizing: border-box;
+  }
+
+  &__stars {
+    display: flex;
+  
+    &_star {
+      background-image: url("/images/star.svg");
+      background-repeat: no-repeat;
+      background-position: center;
+      margin-right: 4px;
+      height: 16px;
+      width: 16px;
+    }
+  
+    &_star:nth-child(5) {
+      margin-right: 0;
+    }
+  
+    &--stars_1 div:nth-child(0),
+    &--stars_2 div:nth-child(n+1):nth-child(-n+2),
+    &--stars_3 div:nth-child(n+1):nth-child(-n+3),
+    &--stars_4 div:nth-child(n+1):nth-child(-n+4),
+    &--stars_5 div:nth-child(n+1):nth-child(-n+5) {
+      background-image: url("/images/star-active.svg");
+      background-repeat: no-repeat;
+      background-position: center;
+    }
+  }
+
+  &__reviev {
+    font-family: "Roboto", sans-serif;
+    color: $mainColor;
+    font-size: 10px;
+    font-weight: 400;
+  }
+
+  &__descr {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: space-between;
+    flex-direction: column;
+    align-items: center;
+    padding: 0 16px 16px;
+    width: 100%;
+  }
+
+  &__modelName {
+    font-family: "Roboto", sans-serif;
+    padding-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $mainColor;
+  }
+  
+  &__modelCode {
+    font-family: "Roboto", sans-serif;
```

Comment:
Just add font-family to the html tag and that's it. Other elements just shall inherit it